### PR TITLE
Banner: nginx 20 years.

### PR DESCRIPTION
**Pull request by y82**
Will be published on Oct. 04, then reverted back to the current banner.

### CODE REVIEW

File: xml/banner.xml [hunk 1]
```diff
@@ -2,9 +2,8 @@
 
 <banner>
 
-Have you heard?
-<link url="https://nginx.org/en/">nginx</link> has moved to
-<link url="https://github.com/nginx/nginx/">GitHub</link>.
-<link url="https://mailman.nginx.org/pipermail/nginx/2024-September/R7KBUUNTG3B2QAAMPS6M7YDOSVCMOZD7.html">Read more</link>.
+Celebrating 20 years of nginx — from the
+<link url="https://hg.nginx.org/nginx/rev/551102312e19">first public version</link>
```
Comment (lcrilly) [inline xml/banner.xml:6]: There will be a celebratory blog post on blog.nginx.org - suggest linking to that instead.

Comment (pluknet) [reply xml/banner.xml:6]: referencing mercurial commit looks awkward after moving to git

Comment (y82) [reply xml/banner.xml]: 20 years ago it was mercurial so I think it's ok here.

File: xml/banner.xml [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
 <banner>
 
-Have you heard?
-<link url="https://nginx.org/en/">nginx</link> has moved to
-<link url="https://github.com/nginx/nginx/">GitHub</link>.
-<link url="https://mailman.nginx.org/pipermail/nginx/2024-September/R7KBUUNTG3B2QAAMPS6M7YDOSVCMOZD7.html">Read more</link>.
+Celebrating <link url="https://hg.nginx.org/nginx/rev/551102312e19">20 years</link>
+of nginx!
+Read about our journey and milestones in the
+<link url="https://blog.nginx.org/blog/celebrating-20-years-of-nginx">latest blog</link>.
```
Comment (pluknet) [inline xml/banner.xml:8]:
```suggestion
Celebrating <link url="https://blog.nginx.org/blog/celebrating-20-years-of-nginx"">20 years</link>
of nginx!
Read about our journey and milestones.
```

File: xml/banner.xml (hunk 1 above)
Comment (y82) [reply xml/banner.xml]: ok, link changed to git: https://github.com/nginx/nginx/commit/0e8348c500bac5303a3b7f2ffee4b7ded05558ff

**Review by pluknet — APPROVED** [review]

**Pull request merged**